lib/swagger-connector.js

```diff
@@ -105,7 +105,7 @@ SwaggerConnector.prototype.connect = function(cb) {
     }
 
     var client = new SwaggerClient({
-      url: self.url,
+      url: self.settings.url,
```

Comment:
I find this change suspicious. What's `self.url` used for after this change? Should we remove it completely? I am concerned there was a reason for its existence and this change may be breaking.

@gunjpan could you please comment?